CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ find_package(catkin REQUIRED
     moveit_ros_planning_interface
     pluginlib
     geometric_shapes
+    moveit_visual_tools
```

Comment:
> Assuming it is wanted, I've decided yes I'm happy to move it to ros-planning. I'll open a issue for it to discuss further.

Awesome! Please do that. It clearly looks like a valuable addition.

Concerning `rviz_visual_tools`, the obvious candidate is https://github.com/ros-visualization .
I don't know what they think of it there.
At first glance it looks like your Swiss army knife and maybe does a bit too much. (at least by now there are other packages for converting Eigen and tf messages for example..)

We could either include this in ros-planning too, if ros-visualization is no option, or you keep it in your personal repo, as this is not "planning" related.